test/AutoDiff/differentiable_func_runtime.swift

```diff
@@ -0,0 +1,28 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+#if os(macOS)
+import Darwin.C
+#else
+import Glibc
+#endif
+
+var DifferentiableFunctionTests = TestSuite("DifferentiableFunctions")
+
+/*
+// FIXME(TF-123): `@differentiable` function thunking with opaque
```

Comment:
Currently, testing isn't possible because reproducers fail SIL verification.
Added negative test, to be enabled when [TF-123](https://bugs.swift.org/projects/TF/issues/TF-123) is resolved.

Reply:
If you want to test it, I think this will work:
```
// RUN: not --crash %target-swift-frontend -emit-sil %s 2>&1 | %FileCheck %s
// REQUIRES: asserts

<some code that fails SILVerifier>

// CHECK: <the expected verification failure>
```

Like this: https://github.com/apple/swift/pull/21367/files#diff-3614888b069c19459a9fdf11f2b169bc

Reply:
Thanks! Done in https://github.com/apple/swift/pull/25803/commits/cd49aa1acee6f00673e7ca1634b399d789727a74.